README.md

```diff
@@ -137,26 +166,41 @@ The Azure Key Vault Provider offers four modes for accessing a Key Vault instanc
 
 > Supported with linux and windows
 
-1. Add your service principal credentials as a Kubernetes secrets accessible by the Secrets Store CSI driver.
+1. Add your service principal credentials as a Kubernetes secrets accessible by the Secrets Store CSI driver. If using AKS you can learn about [service principals in AKS here.](https://docs.microsoft.com/azure/aks/kubernetes-service-principal)
```

Comment:
This is great! Can you also add https://docs.microsoft.com/en-us/azure/aks/use-managed-identity for AKS for the other two options: system assigned and user assigned identity.

Reply:
good idea, adding for system assigned and adding a note AKS has a gap to support UMI.